.github/workflows/deploy_docs.yml

```diff
@@ -8,7 +8,8 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
-      - run: rustup component add rustfmt clippy rust-src rustc-dev llvm-tools-preview
+      - run: sh <(curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs) -y
+        shell: bash
```

Comment:
is this needed? I guess it's a little hard to test since deploy_docs doesn't run until it's pushed to main

Reply:
Oh wait I forgot to remove it from `deploy_docs.yml` :facepalm:.

As for `rustfmt` and `clippy`, they're not part of the `default` profile AFAICT, so unless we want to add them to `rust-toolchain`, they won't be available.